src/cclib/io/ccio.py

```diff
@@ -355,29 +356,33 @@ def _determine_output_format(outputtype, outputdest):
     #  1. outputtype
```

Comment:
This is an artefact from while I was coding; should not have been a part of the code. I'll remove all these in a commit.